lib/cassanity/executors/cql_rb.rb

```diff
@@ -185,7 +184,7 @@ def call(args = {})
             end
 
             # Select the correct keyspace before executing the CQL query
-            if command != :keyspace_create && (keyspace_name = arguments[:keyspace_name])
+            if !command.to_s['keyspace'] && (keyspace_name = arguments[:keyspace_name])
```

Comment:
Do we need to convert this to_s? Isn't it a symbol currently? Converting would mean creating a lot more strings.